Assets/VRTK/Scripts/VRTK_Hip_Tracking.cs

```diff
@@ -0,0 +1,66 @@
+﻿// Hip Tracking|Scripts|0135
+namespace VRTK
+{
+	using UnityEngine;
+	using System.Collections;
+
+
+	/// <summary>
+	/// Hip Tracking attempts to reasonably track hip position in the absence of a hip position sensor
+	/// </summary>
+	/// <remarks>
+	///
```

Comment:
Don't need this blank line, a paragraph is already added when splitting summary and remarks